deps/linenoise/linenoise.c

```diff
@@ -134,6 +134,8 @@ static int atexit_registered = 0; /* Register atexit just 1 time. */
 static int history_max_len = LINENOISE_DEFAULT_HISTORY_MAX_LEN;
 static int history_len = 0;
 static char **history = NULL;
+static int *history_sensitive = NULL; /* An array records whether each line in
+                                       * history is sensitive. */
```

Comment:
An array of char would be enough to just store 1 or 0. But I guess memory is not an issue here, so it doesn't matter?